modules/services/syncthing.nix

```diff
@@ -118,7 +122,11 @@ in {
           Unit = {
             Description = "syncthingtray";
             Requires = [ "tray.target" ];
```

Comment:
Maybe `syncthing.service` should be added here to so that the tray only starts with the other service and dies when the other service is unavailable?